testbed/eks/README.md

```diff
@@ -0,0 +1,14 @@
+# Welcome to your CDK TypeScript project!
+
+You should explore the contents of this project. It demonstrates a CDK app with an instance of a stack (`TestbedStack`)
+
+The `cdk.json` file tells the CDK Toolkit how to execute your app.
+
+## Useful commands
+
+ * `npm run build`   compile typescript to js
+ * `npm run watch`   watch for changes and compile
+ * `npm run test`    perform the jest unit tests
+ * `cdk deploy`      deploy this stack to your default AWS account/region
+ * `cdk diff`        compare deployed stack with current state
+ * `cdk synth`       emits the synthesized CloudFormation template
```

Comment:
Do we need to run these commands to deploy the testbed, or are these more of good to know commands?

Reply:
Yes, we need them.